engines/query-sparql-file/package.json

```diff
@@ -163,7 +163,6 @@
     "@comunica/actor-rdf-update-quads-rdfjs-store": "^2.6.9",
     "@comunica/bus-http-invalidate": "^2.6.8",
     "@comunica/bus-query-operation": "^2.6.8",
-    "@comunica/config-query-sparql": "^2.6.0",
```

Comment:
This one should definitely be present, as it will be used when instantiating an engine dynamically from the config file using the query engine factory.

Reply:
It's not present in any js files. The only time `@comunica/config-query-sparql` is mentioned (in this directory) is in the `config-default.json` and `config-browser.json` files.

Reply:
```
{
  "@context": [
    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/config-query-sparql/^2.0.0/components/context.jsonld"
  ],
  "import": [
    "ccqs:config/config-file.json"
  ]
}
```

Reply:
We'll need a workaround for it then, as it will otherwise cause errors during dependency injection.